multiversx_sdk_core/custom_tokens.py

```diff
@@ -0,0 +1,49 @@
+from multiversx_sdk_core.codec import decode_unsigned_number
+from multiversx_sdk_core.errors import InvalidTokenIdentifierError
+
+
+class Token:
+    def __init__(self, identifier: str = "", nonce: int = 0) -> None:
+        self.identifier = identifier
+        self.nonce = nonce
+
+
+class TokenTransfer:
+    def __init__(self, token: Token, amount: int) -> None:
+        """`amount` should always be in atomic units: 1 EGLD = 1000000000000000000"""
+        self.token = token
+        self.amount = amount
+
+
+# the rest of the methods will be implemented in a future PR
```

Comment:
In the future PR, we should add unit tests, as well.

Reply:
Indeed, will add tests.